README.md

```diff
@@ -79,18 +79,18 @@ Download [the JARs][]. Or for Maven with non-Android, add to your `pom.xml`:
 
 Or for Gradle with non-Android, add to your dependencies:
 ```gradle
-runtimeOnly 'io.grpc:grpc-netty-shaded:1.66.0'
-implementation 'io.grpc:grpc-protobuf:1.66.0'
-implementation 'io.grpc:grpc-stub:1.66.0'
+runtimeOnly 'io.grpc:grpc-netty-shaded:1.67.0'
+implementation 'io.grpc:grpc-protobuf:1.67.0'
+implementation 'io.grpc:grpc-stub:1.67.0'
 compileOnly 'org.apache.tomcat:annotations-api:6.0.53' // necessary for Java 9+
 ```
 
 For Android client, use `grpc-okhttp` instead of `grpc-netty-shaded` and
 `grpc-protobuf-lite` instead of `grpc-protobuf`:
 ```gradle
-implementation 'io.grpc:grpc-okhttp:1.66.0'
-implementation 'io.grpc:grpc-protobuf-lite:1.66.0'
-implementation 'io.grpc:grpc-stub:1.66.0'
+implementation 'io.grpc:grpc-okhttp:1.67.0'
+implementation 'io.grpc:grpc-protobuf-lite:1.67.0'
+implementation 'io.grpc:grpc-stub:1.67.0'
 compileOnly 'org.apache.tomcat:annotations-api:6.0.53' // necessary for Java 9+
 ```
 
```

Comment:
"the JARs" link below also has a reference to the old 1.66.0. You may want to apply this as-is and just have that minor issue in the release, but then fix it on master and the v1.67.x branch after-the-fact.

Reply:
Or should I do a new commit making it 1.67.x?

Reply:
If you just add another commit, it won't be in the release anyway. You could delete your local v1.66.0 tag and fix the "Update README etc to reference 1.67.0" commit, but I sort of expect that to lead to far more problems for your first release.

Reply:
Alright! Sounds good.